caps/types.go

```diff
@@ -23,28 +23,37 @@ import (
 	"fmt"
 )
 
-// Type is the name of a capability type.
-type Type string
+// Type describes a group of interchangeable capabilities with common features.
+// Types are managed centrally and act as a contract between system builders,
+// application developers and end users.
+type Type struct {
+	// Name is a key that identifies the capability type. It must be unique
+	// within the whole OS. The name forms a part of the stable system API.
+	Name string
+}
 
-const (
+var (
 	// FileType is a basic capability vaguely expressing access to a specific
 	// file. This single capability  type is here just to help bootstrap
 	// the capability concept before we get to load capability interfaces
 	// from YAML.
-	FileType Type = "file"
+	FileType = &Type{"file"}
+	// testType is only meant for testing. It is not useful in any way except
+	// that it offers an simple capability type that will happily validate.
+	testType = &Type{"test"}
```

Comment:
Idle thought, 'testType = &Type{Name: "test"}' but I guess its not important given that there is only a single member in the struct :)